build-ps/build-binary.sh

```diff
@@ -287,6 +287,85 @@ fi
     fi
 )
 
+# Patch needed libraries
+(
+    cd "$INSTALLDIR/usr/local/$PRODUCT_FULL"
+    if [ ! -d lib/private ]; then
+        mkdir -p lib/private
+    fi
+
+    LIBLIST="libcrypto.so libssl.so libreadline.so libtinfo.so libsasl2.so libcurl.so libldap liblber libssh.so libbrotlidec.so libbrotlicommon.so libgssapi_krb5.so libkrb5.so libkrb5support.so libk5crypto.so librtmp.so libgssapi.so libhcrypto.so libheimbase.so libcrypt.so libwind.so libhx509.so libasn1.so libheimntlm.so libroken.so"
```

Comment:
Who will manage this list of libs(add/delete)?

Reply:
There's no way to do it in automatic way, at least I haven't managed to discover.

And I think it's better to discuss it

Reply:
I am asking about it because we will not know if something is changed.

Reply:
Your statement is true

I have few scripts which I made for myself for testing purpose, like check for "not found" libs in ldd of ELF

Plus, the logic of script uses `$LIBLIST` as market variable, it automatically gather versions of needed libs, only one thing is left: how to track new dependencies

Reply:
@hors @EvgeniyPatlan Tests reference: https://rel.cd.percona.com/job/percona-server-80/